framework/wazuh/core/exception.py

```diff
@@ -437,6 +437,9 @@ class WazuhException(Exception):
         # External services
         2100: {'message': 'Error in CTI service request'},
 
+        # Indexer
+        2200: {'message': 'Could not initialize Indexer connection'},
```

Comment:
Not really necessary, but to be consistent with other error messages

```suggestion
        2200: {'message': 'Could not connect to the indexer'},
```